doc-content/enterprise-only/cases/case-management-variable-tags-con.adoc

```diff
@@ -0,0 +1,65 @@
+[id='case-management-variable-tags-con']
+= Variable tags
+Variables store data that is used during runtime. For greater control over variable behavior, you can tag case variables and local variables in the BPMN case file. Tags are simple string values that you add as metadata to a specific variable.
+
+{PRODUCT} supports the following tags for case and local variables:
+
+* *required*: Sets the variable as a requirement in order to start a case. If a case starts without the required variable, {PRODUCT} generates a `VariableViolationException` error.
+* *readonly*: Indicates that the variable is for informational purposes only and can be set only once during case execution. If the value of a read-only variable is modified at any time, {PRODUCT} generates a `VariableViolationException` error.
+* *restricted*: A tag that is used with the `VariableGuardProcessEventListener` to indicate that permission is granted to modify the variable based on the existing role. The *restricted* tag can be replaced by any other tag name if using the second constructor that passes the new tag name.
+
+`VariableGuardProcessEventListener` is extended from `DefaultProcessEventListener` and supports two different constructors:
```

Comment:
I would say:
The `VariableGuardProcessEventListener`class is extended from DefaultProcessEventListener class...

Reply:
Updated as suggested.